open_academy/i18n/es_MX.po

```diff
@@ -0,0 +1,250 @@
+# Translation of Odoo Server.
```

Comment:
When we don't know what localization the user has for their language (if they are from Spain, Mexico, Colombia), we use the es.po file. This will allow the user to use any localization for the Spanish language, it will have our translation.

Please rename this file to es.po